packages/mcp-provider-dx-core/package.json

```diff
@@ -56,25 +48,29 @@
   },
   "devDependencies": {
     "@salesforce/mcp-test-client": "0.0.1",
-    "@modelcontextprotocol/inspector": "^0.15.0",
+    "@salesforce/dev-config": "^4.3.2",
     "@salesforce/cli-plugins-testkit": "^5.3.39",
-    "@salesforce/dev-scripts": "11.0.2",
     "@types/node": "^22.16.5",
+    "wireit": "^0.14.12",
     "eslint": "^8.57.1",
+    "mocha": "11.7.2",
+    "chai": "^4.3.10",
+    "@types/mocha": "^10.0.10",
+    "@types/chai": "^4.3.14",
+    "@types/sinon": "^10.0.20",
+    "sinon": "10.0.0",
+    "nyc": "^17.0.0",
     "eslint-config-salesforce-license": "^1.0.1",
-    "eslint-plugin-sf-plugin": "^1.20.26",
+    "eslint-config-salesforce-typescript": "4.0.1",
+    "@salesforce/prettier-config": "^0.0.3",
```

Comment:
all these deps were being installed via dev-scripts, need to add them manually now that it's gone.